react/Icon/Readme.md

```diff
@@ -600,3 +602,12 @@ const handleInputRangeChange = ev => {
   </div>
 </div>
 ```
+
+## Add an Icon
+
+If you want to add a new icon, you must follow these steps:
+
+* Add the SVG in the `assets/icons` folder
+* Generate the react component, simply by running `scripts/generate-svgr-icon.sh`
+* Update this README by adding your icon in the list
+* Update the tests by running `yarn test --updateSnapshot`
```

Comment:
This should be in the README, not in the example of the styleguide.

Reply:
Well, this is a readme :smile: But sure, I'll move it to the main one

Reply:
Done with e04f839